src/main/src/main/java/org/geoserver/catalog/LayerGroupInfo.java

```diff
@@ -88,6 +88,19 @@ public Integer getCode() {
     void setMode( Mode mode );    
 
     /**
+     * Get whether the layer group is forced to be not queryable and hence can not be subject of a GetFeatureInfo request.
+     * <p>
+     * Default is FALSE
+     * </p>
+     */
+    Boolean isNotQueryableEnabled();
```

Comment:
Ah right, you have a point... let's call it isQueryDisabled() then,  pointing out in the javadoc that the default behavior is to respect the layers own queriable flags (isNotQueryableEnabled is not readable english).